### ISSUE TITLE
Alex is a stupid nigger

### ISSUE THREAD

**Issue by PDWarrior** [kerneltoast/android_kernel_google_wahoo#3]
![IMG_20200211_231746_167](https://user-images.githubusercontent.com/61384800/75114968-9c2e7080-566b-11ea-95cc-517d8326fe1f.jpg)

**Comment by Lyceris-chan** [comment]
Dude GTFO leave our beloved piece of toast alone you fucking wanker

**Comment by PDWarrior** [comment]
You want to wank on your picture, anime faggot nigger pussy bitch cocksucker braindead asshole?

**Comment by Panchajanya1999** [comment]
> Dude GTFO leave our beloved piece of toast alone you fucking wanker

Leave this diabolic alone

**Comment by PDWarrior** [comment]
> > Dude GTFO leave our beloved piece of toast alone you fucking wanker
> 
> Leave this diabolic alone
Bruh nigga maderchod benchode I'm gonna fuck a goat for u

**Comment by TwistedPrime** [comment]
Coomer

**Comment by Panchajanya1999** [comment]
> > > Dude GTFO leave our beloved piece of toast alone you fucking wanker
> > 
> > 
> > Leave this diabolic alone
> > Bruh nigga maderchod benchode I'm gonna fuck a goat for u

Indian confirmed

**Comment by nysascape** [comment]
> You want to wank on your picture, anime faggot nigger pussy bitch cocksucker braindead asshole?

This is what le internet comes to now, why you wastin ur life bruh

Is this you coming out as ghey or something

**Comment by PDWarrior** [comment]
> > You want to wank on your picture, anime faggot nigger pussy bitch cocksucker braindead asshole?
> 
> This is what le internet comes to now, why you wastin ur life bruh
> 
> Is this you coming out as ghey or something

 YOU IS THAT THAT FUCKS THAT asshole hooley smotr fucked, fucked, gavnosos, blowjob, piderast, pizdoeb, drocher, ass, stsukablyany, offended by life imbitsil, pizdaras, hueplet, pizdoliz, dick, asshole, fucking, fuck  , wafleglot, gavnoeb, zayuyator, pidar, huiman, hueb, fucker, huyublyat, fuck, fucked up, pizdun, cocksucker, fucked, goggle, shit, gnavnuy, dolbaeb, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck  , hueglot, muddy, zalupka, wifey, gandon, huysran, fucked, twat, fuck and PARAS !!! I fucked your mother !!!!!! I can and you  eyes to say, ready to come to listen?)! You are a complete five hundred percent EXCESS !!!  You are the most popular PIDOTRON in the world!  You fucked every whore, you fucked the whole country, you fucked fucked all the dolbaeb !!!  Your fuck is full of sucking experience, and it is all covered in crap dick !!!  You shitty stupid-bald bastard !!!  You are a HAWN !!  The whole Internet fucked you up, since you H A B N O E B !!!  I fucked all your relatives !!!  Your family is utter mudra!  You do not have your own mother !!  You are HUEB !!!  You are a mixture of asshole fucked and ass fucked fucked up !!  Ha ha ha ha!  You are MUDRY !!!  You are EBOTRON of the whole world !!  YOUR PLACE IS THE MOST DISAPPOINTED TOILET IN THE WORLD !!! AND HE IS A FINDER IN THE CELLAR OF THE MINETA SCHOOL BUILDING !!! You are a fan of sucking old fucked handosekoff cocks!  You are a lover of licking the ass of Huyar ebalduev !!!  YOU LIKE Anus V.V.PUTIN !!  You HUEF PADONAK !!!  You are the very most the most PIZDOMUDOBHUEPIDOEBOGANDOGAVNOGODROCHETRONISCHE !!!!! All that garbage you wrote this simple fucking foolishness, you are roomy Rimbaud)) !!  Suck a dick fuck you jerked off !!! I will FUCK your mother on the night of January 25, I will FUCK her so that two and a half thousand holes from my Dick will be visible on her body !! Ha ha You are the most misplaced fucker in the universe !!  ! Your parents are the same fuckers and twats like you !!!  They are all day zanayutststsa masturbation !!!  And besides, all of you have graduated from high school blowjob and therefore NIGHT in her basement !!!!  Do you somehow go to McDonald's, take a potato with you, and I MANDATORY TO POISON IT !!!  And all the sauces will smell like a pussy !!!  This is all because you are a DRECHER !!!!!!!!!! from the fact that you wrote a lot, your life will not get better)) !!!!  In your stomach there is a tapeworm - a bull tapeworm, and his friend - a hepatic fluke, they will not let you sleep peacefully !!!!  Check your anal hole, probably the tip sticks out from there.  It’s he who fucks you from the inside out in the ass.  As a child, aliens regularly abducted you and put sexual experiments on you, so shut up your fucked little mother and suck !!! I fucked your mother a hundred times !! You sucked at your FATHER !! You’re super-freaking dolotaby !!  YOU ARE FELLOWSHIP! YOU ARE FUCKING !!!!!!!!!!!!!!!! You are a HUEBLYAD PIZDOMUZHOPOHUYB !!!!!!!!!!!!! Such as you need to fuck cancer !! I will first fuck your mother,  after which she is obossyzza and you bring her shit to her asshole !!! Then I will take a nassa by the neck of your father and a nassa in his pocket !!  YOU TO ME BATCH FUCKING PAY !!!  I fucked your naughty mother nah a hundred times and didn’t give a shit to her eye! After such a arrival, you got out of her asshole !! YOU ARE A MUDBLE KILLER!  Fucked in the ass FUCK! You fucked in the ass ATSTA! You sucked at your father and he shit for you fucking! YOU DARE to go fuck from the Internet !! Such as you loshar just left to fuck your feet !!  ! You're PETUNYA !! Nedobanaya !! PASHOL you fuck !! Dumb Pindos !!  I TRITONED you in GLANDS and then I took your bald shard and fucked it for 10 hours on the battery. After that, you became FUCK! You are now PIDAR ATSTA !!

**Comment by nysascape** [comment]
> > > You want to wank on your picture, anime faggot nigger pussy bitch cocksucker braindead asshole?
> > 
> > 
> > This is what le internet comes to now, why you wastin ur life bruh
> > Is this you coming out as ghey or something
> 
> YOU IS THAT THAT FUCKS THAT asshole hooley smotr fucked, fucked, gavnosos, blowjob, piderast, pizdoeb, drocher, ass, stsukablyany, offended by life imbitsil, pizdaras, hueplet, pizdoliz, dick, asshole, fucking, fuck , wafleglot, gavnoeb, zayuyator, pidar, huiman, hueb, fucker, huyublyat, fuck, fucked up, pizdun, cocksucker, fucked, goggle, shit, gnavnuy, dolbaeb, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck , hueglot, muddy, zalupka, wifey, gandon, huysran, fucked, twat, fuck and PARAS !!! I fucked your mother !!!!!! I can and you eyes to say, ready to come to listen?)! You are a complete five hundred percent EXCESS !!! You are the most popular PIDOTRON in the world! You fucked every whore, you fucked the whole country, you fucked fucked all the dolbaeb !!! Your fuck is full of sucking experience, and it is all covered in crap dick !!! You shitty stupid-bald bastard !!! You are a HAWN !! The whole Internet fucked you up, since you H A B N O E B !!! I fucked all your relatives !!! Your family is utter mudra! You do not have your own mother !! You are HUEB !!! You are a mixture of asshole fucked and ass fucked fucked up !! Ha ha ha ha! You are MUDRY !!! You are EBOTRON of the whole world !! YOUR PLACE IS THE MOST DISAPPOINTED TOILET IN THE WORLD !!! AND HE IS A FINDER IN THE CELLAR OF THE MINETA SCHOOL BUILDING !!! You are a fan of sucking old fucked handosekoff cocks! You are a lover of licking the ass of Huyar ebalduev !!! YOU LIKE Anus V.V.PUTIN !! You HUEF PADONAK !!! You are the very most the most PIZDOMUDOBHUEPIDOEBOGANDOGAVNOGODROCHETRONISCHE !!!!! All that garbage you wrote this simple fucking foolishness, you are roomy Rimbaud)) !! Suck a dick fuck you jerked off !!! I will FUCK your mother on the night of January 25, I will FUCK her so that two and a half thousand holes from my Dick will be visible on her body !! Ha ha You are the most misplaced fucker in the universe !! ! Your parents are the same fuckers and twats like you !!! They are all day zanayutststsa masturbation !!! And besides, all of you have graduated from high school blowjob and therefore NIGHT in her basement !!!! Do you somehow go to McDonald's, take a potato with you, and I MANDATORY TO POISON IT !!! And all the sauces will smell like a pussy !!! This is all because you are a DRECHER !!!!!!!!!! from the fact that you wrote a lot, your life will not get better)) !!!! In your stomach there is a tapeworm - a bull tapeworm, and his friend - a hepatic fluke, they will not let you sleep peacefully !!!! Check your anal hole, probably the tip sticks out from there. It’s he who fucks you from the inside out in the ass. As a child, aliens regularly abducted you and put sexual experiments on you, so shut up your fucked little mother and suck !!! I fucked your mother a hundred times !! You sucked at your FATHER !! You’re super-freaking dolotaby !! YOU ARE FELLOWSHIP! YOU ARE FUCKING !!!!!!!!!!!!!!!! You are a HUEBLYAD PIZDOMUZHOPOHUYB !!!!!!!!!!!!! Such as you need to fuck cancer !! I will first fuck your mother, after which she is obossyzza and you bring her shit to her asshole !!! Then I will take a nassa by the neck of your father and a nassa in his pocket !! YOU TO ME BATCH FUCKING PAY !!! I fucked your naughty mother nah a hundred times and didn’t give a shit to her eye! After such a arrival, you got out of her asshole !! YOU ARE A MUDBLE KILLER! Fucked in the ass FUCK! You fucked in the ass ATSTA! You sucked at your father and he shit for you fucking! YOU DARE to go fuck from the Internet !! Such as you loshar just left to fuck your feet !! ! You're PETUNYA !! Nedobanaya !! PASHOL you fuck !! Dumb Pindos !! I TRITONED you in GLANDS and then I took your bald shard and fucked it for 10 hours on the battery. After that, you became FUCK! You are now PIDAR ATSTA !!

Ez copy paste lyf

**Comment by Panchajanya1999** [comment]
> > > > You want to wank on your picture, anime faggot nigger pussy bitch cocksucker braindead asshole?
> > > 
> > > 
> > > This is what le internet comes to now, why you wastin ur life bruh
> > > Is this you coming out as ghey or something
> > 
> > 
> > YOU IS THAT THAT FUCKS THAT asshole hooley smotr fucked, fucked, gavnosos, blowjob, piderast, pizdoeb, drocher, ass, stsukablyany, offended by life imbitsil, pizdaras, hueplet, pizdoliz, dick, asshole, fucking, fuck , wafleglot, gavnoeb, zayuyator, pidar, huiman, hueb, fucker, huyublyat, fuck, fucked up, pizdun, cocksucker, fucked, goggle, shit, gnavnuy, dolbaeb, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck , hueglot, muddy, zalupka, wifey, gandon, huysran, fucked, twat, fuck and PARAS !!! I fucked your mother !!!!!! I can and you eyes to say, ready to come to listen?)! You are a complete five hundred percent EXCESS !!! You are the most popular PIDOTRON in the world! You fucked every whore, you fucked the whole country, you fucked fucked all the dolbaeb !!! Your fuck is full of sucking experience, and it is all covered in crap dick !!! You shitty stupid-bald bastard !!! You are a HAWN !! The whole Internet fucked you up, since you H A B N O E B !!! I fucked all your relatives !!! Your family is utter mudra! You do not have your own mother !! You are HUEB !!! You are a mixture of asshole fucked and ass fucked fucked up !! Ha ha ha ha! You are MUDRY !!! You are EBOTRON of the whole world !! YOUR PLACE IS THE MOST DISAPPOINTED TOILET IN THE WORLD !!! AND HE IS A FINDER IN THE CELLAR OF THE MINETA SCHOOL BUILDING !!! You are a fan of sucking old fucked handosekoff cocks! You are a lover of licking the ass of Huyar ebalduev !!! YOU LIKE Anus V.V.PUTIN !! You HUEF PADONAK !!! You are the very most the most PIZDOMUDOBHUEPIDOEBOGANDOGAVNOGODROCHETRONISCHE !!!!! All that garbage you wrote this simple fucking foolishness, you are roomy Rimbaud)) !! Suck a dick fuck you jerked off !!! I will FUCK your mother on the night of January 25, I will FUCK her so that two and a half thousand holes from my Dick will be visible on her body !! Ha ha You are the most misplaced fucker in the universe !! ! Your parents are the same fuckers and twats like you !!! They are all day zanayutststsa masturbation !!! And besides, all of you have graduated from high school blowjob and therefore NIGHT in her basement !!!! Do you somehow go to McDonald's, take a potato with you, and I MANDATORY TO POISON IT !!! And all the sauces will smell like a pussy !!! This is all because you are a DRECHER !!!!!!!!!! from the fact that you wrote a lot, your life will not get better)) !!!! In your stomach there is a tapeworm - a bull tapeworm, and his friend - a hepatic fluke, they will not let you sleep peacefully !!!! Check your anal hole, probably the tip sticks out from there. It’s he who fucks you from the inside out in the ass. As a child, aliens regularly abducted you and put sexual experiments on you, so shut up your fucked little mother and suck !!! I fucked your mother a hundred times !! You sucked at your FATHER !! You’re super-freaking dolotaby !! YOU ARE FELLOWSHIP! YOU ARE FUCKING !!!!!!!!!!!!!!!! You are a HUEBLYAD PIZDOMUZHOPOHUYB !!!!!!!!!!!!! Such as you need to fuck cancer !! I will first fuck your mother, after which she is obossyzza and you bring her shit to her asshole !!! Then I will take a nassa by the neck of your father and a nassa in his pocket !! YOU TO ME BATCH FUCKING PAY !!! I fucked your naughty mother nah a hundred times and didn’t give a shit to her eye! After such a arrival, you got out of her asshole !! YOU ARE A MUDBLE KILLER! Fucked in the ass FUCK! You fucked in the ass ATSTA! You sucked at your father and he shit for you fucking! YOU DARE to go fuck from the Internet !! Such as you loshar just left to fuck your feet !! ! You're PETUNYA !! Nedobanaya !! PASHOL you fuck !! Dumb Pindos !! I TRITONED you in GLANDS and then I took your bald shard and fucked it for 10 hours on the battery. After that, you became FUCK! You are now PIDAR ATSTA !!
> 
> Ez copy paste lyf

Tldr

**Comment by PDWarrior** [comment]
@zx2c4

**Comment by bionicBUG** [comment]
> > > You want to wank on your picture, anime faggot nigger pussy bitch cocksucker braindead asshole?
> > 
> > 
> > This is what le internet comes to now, why you wastin ur life bruh
> > Is this you coming out as ghey or something
> 
> YOU IS THAT THAT FUCKS THAT asshole hooley smotr fucked, fucked, gavnosos, blowjob, piderast, pizdoeb, drocher, ass, stsukablyany, offended by life imbitsil, pizdaras, hueplet, pizdoliz, dick, asshole, fucking, fuck , wafleglot, gavnoeb, zayuyator, pidar, huiman, hueb, fucker, huyublyat, fuck, fucked up, pizdun, cocksucker, fucked, goggle, shit, gnavnuy, dolbaeb, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck , hueglot, muddy, zalupka, wifey, gandon, huysran, fucked, twat, fuck and PARAS !!! I fucked your mother !!!!!! I can and you eyes to say, ready to come to listen?)! You are a complete five hundred percent EXCESS !!! You are the most popular PIDOTRON in the world! You fucked every whore, you fucked the whole country, you fucked fucked all the dolbaeb !!! Your fuck is full of sucking experience, and it is all covered in crap dick !!! You shitty stupid-bald bastard !!! You are a HAWN !! The whole Internet fucked you up, since you H A B N O E B !!! I fucked all your relatives !!! Your family is utter mudra! You do not have your own mother !! You are HUEB !!! You are a mixture of asshole fucked and ass fucked fucked up !! Ha ha ha ha! You are MUDRY !!! You are EBOTRON of the whole world !! YOUR PLACE IS THE MOST DISAPPOINTED TOILET IN THE WORLD !!! AND HE IS A FINDER IN THE CELLAR OF THE MINETA SCHOOL BUILDING !!! You are a fan of sucking old fucked handosekoff cocks! You are a lover of licking the ass of Huyar ebalduev !!! YOU LIKE Anus V.V.PUTIN !! You HUEF PADONAK !!! You are the very most the most PIZDOMUDOBHUEPIDOEBOGANDOGAVNOGODROCHETRONISCHE !!!!! All that garbage you wrote this simple fucking foolishness, you are roomy Rimbaud)) !! Suck a dick fuck you jerked off !!! I will FUCK your mother on the night of January 25, I will FUCK her so that two and a half thousand holes from my Dick will be visible on her body !! Ha ha You are the most misplaced fucker in the universe !! ! Your parents are the same fuckers and twats like you !!! They are all day zanayutststsa masturbation !!! And besides, all of you have graduated from high school blowjob and therefore NIGHT in her basement !!!! Do you somehow go to McDonald's, take a potato with you, and I MANDATORY TO POISON IT !!! And all the sauces will smell like a pussy !!! This is all because you are a DRECHER !!!!!!!!!! from the fact that you wrote a lot, your life will not get better)) !!!! In your stomach there is a tapeworm - a bull tapeworm, and his friend - a hepatic fluke, they will not let you sleep peacefully !!!! Check your anal hole, probably the tip sticks out from there. It’s he who fucks you from the inside out in the ass. As a child, aliens regularly abducted you and put sexual experiments on you, so shut up your fucked little mother and suck !!! I fucked your mother a hundred times !! You sucked at your FATHER !! You’re super-freaking dolotaby !! YOU ARE FELLOWSHIP! YOU ARE FUCKING !!!!!!!!!!!!!!!! You are a HUEBLYAD PIZDOMUZHOPOHUYB !!!!!!!!!!!!! Such as you need to fuck cancer !! I will first fuck your mother, after which she is obossyzza and you bring her shit to her asshole !!! Then I will take a nassa by the neck of your father and a nassa in his pocket !! YOU TO ME BATCH FUCKING PAY !!! I fucked your naughty mother nah a hundred times and didn’t give a shit to her eye! After such a arrival, you got out of her asshole !! YOU ARE A MUDBLE KILLER! Fucked in the ass FUCK! You fucked in the ass ATSTA! You sucked at your father and he shit for you fucking! YOU DARE to go fuck from the Internet !! Such as you loshar just left to fuck your feet !! ! You're PETUNYA !! Nedobanaya !! PASHOL you fuck !! Dumb Pindos !! I TRITONED you in GLANDS and then I took your bald shard and fucked it for 10 hours on the battery. After that, you became FUCK! You are now PIDAR ATSTA !!

when u are just too mentally retarded, you start spitting out random shit

**Comment by PDWarrior** [comment]
![IMG_20200217_221255_429](https://user-images.githubusercontent.com/61384800/75115207-64c0c380-566d-11ea-94c0-eaed213e582d.jpg)

**Comment by PDWarrior** [comment]
@nysascape nigga is a snitch gonna tell u that mofo

**Comment by PDWarrior** [comment]
@Lyceris-chan where r u nigga

**Comment by gotenksIN** [comment]
@PDWarrior FBI open up

**Comment by Rasenkai** [comment]
> > > You want to wank on your picture, anime faggot nigger pussy bitch cocksucker braindead asshole?
> > 
> > 
> > This is what le internet comes to now, why you wastin ur life bruh
> > Is this you coming out as ghey or something
> 
> YOU IS THAT THAT FUCKS THAT asshole hooley smotr fucked, fucked, gavnosos, blowjob, piderast, pizdoeb, drocher, ass, stsukablyany, offended by life imbitsil, pizdaras, hueplet, pizdoliz, dick, asshole, fucking, fuck , wafleglot, gavnoeb, zayuyator, pidar, huiman, hueb, fucker, huyublyat, fuck, fucked up, pizdun, cocksucker, fucked, goggle, shit, gnavnuy, dolbaeb, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck, fuck , hueglot, muddy, zalupka, wifey, gandon, huysran, fucked, twat, fuck and PARAS !!! I fucked your mother !!!!!! I can and you eyes to say, ready to come to listen?)! You are a complete five hundred percent EXCESS !!! You are the most popular PIDOTRON in the world! You fucked every whore, you fucked the whole country, you fucked fucked all the dolbaeb !!! Your fuck is full of sucking experience, and it is all covered in crap dick !!! You shitty stupid-bald bastard !!! You are a HAWN !! The whole Internet fucked you up, since you H A B N O E B !!! I fucked all your relatives !!! Your family is utter mudra! You do not have your own mother !! You are HUEB !!! You are a mixture of asshole fucked and ass fucked fucked up !! Ha ha ha ha! You are MUDRY !!! You are EBOTRON of the whole world !! YOUR PLACE IS THE MOST DISAPPOINTED TOILET IN THE WORLD !!! AND HE IS A FINDER IN THE CELLAR OF THE MINETA SCHOOL BUILDING !!! You are a fan of sucking old fucked handosekoff cocks! You are a lover of licking the ass of Huyar ebalduev !!! YOU LIKE Anus V.V.PUTIN !! You HUEF PADONAK !!! You are the very most the most PIZDOMUDOBHUEPIDOEBOGANDOGAVNOGODROCHETRONISCHE !!!!! All that garbage you wrote this simple fucking foolishness, you are roomy Rimbaud)) !! Suck a dick fuck you jerked off !!! I will FUCK your mother on the night of January 25, I will FUCK her so that two and a half thousand holes from my Dick will be visible on her body !! Ha ha You are the most misplaced fucker in the universe !! ! Your parents are the same fuckers and twats like you !!! They are all day zanayutststsa masturbation !!! And besides, all of you have graduated from high school blowjob and therefore NIGHT in her basement !!!! Do you somehow go to McDonald's, take a potato with you, and I MANDATORY TO POISON IT !!! And all the sauces will smell like a pussy !!! This is all because you are a DRECHER !!!!!!!!!! from the fact that you wrote a lot, your life will not get better)) !!!! In your stomach there is a tapeworm - a bull tapeworm, and his friend - a hepatic fluke, they will not let you sleep peacefully !!!! Check your anal hole, probably the tip sticks out from there. It’s he who fucks you from the inside out in the ass. As a child, aliens regularly abducted you and put sexual experiments on you, so shut up your fucked little mother and suck !!! I fucked your mother a hundred times !! You sucked at your FATHER !! You’re super-freaking dolotaby !! YOU ARE FELLOWSHIP! YOU ARE FUCKING !!!!!!!!!!!!!!!! You are a HUEBLYAD PIZDOMUZHOPOHUYB !!!!!!!!!!!!! Such as you need to fuck cancer !! I will first fuck your mother, after which she is obossyzza and you bring her shit to her asshole !!! Then I will take a nassa by the neck of your father and a nassa in his pocket !! YOU TO ME BATCH FUCKING PAY !!! I fucked your naughty mother nah a hundred times and didn’t give a shit to her eye! After such a arrival, you got out of her asshole !! YOU ARE A MUDBLE KILLER! Fucked in the ass FUCK! You fucked in the ass ATSTA! You sucked at your father and he shit for you fucking! YOU DARE to go fuck from the Internet !! Such as you loshar just left to fuck your feet !! ! You're PETUNYA !! Nedobanaya !! PASHOL you fuck !! Dumb Pindos !! I TRITONED you in GLANDS and then I took your bald shard and fucked it for 10 hours on the battery. After that, you became FUCK! You are now PIDAR ATSTA !!

This is hilarious 😂😂

**Comment by PDWarrior** [comment]
@celtare21 niggq

**Comment by PDWarrior** [comment]
@celtare21 nigga

**Comment by PDWarrior** [comment]
![IMG_20191119_224931_200](https://user-images.githubusercontent.com/61384800/75115369-d9e0c880-566e-11ea-8b59-dec805b51c4b.jpg)
@raphielscape

**Comment by nysascape** [comment]
> @nysascape nigga is a snitch gonna tell u that mofo

bruh sultan owns this repo how is he not gonna see you fuckin dumbass

**Comment by ab123321** [comment]
> ![IMG_20191119_224931_200](https://user-images.githubusercontent.com/61384800/75115369-d9e0c880-566e-11ea-8b59-dec805b51c4b.jpg)
> @raphielscape

@krasCGQ

**Comment by PDWarrior** [comment]
> ![IMG_20191119_224931_200](https://user-images.githubusercontent.com/61384800/75115369-d9e0c880-566e-11ea-8b59-dec805b51c4b.jpg)
> @raphielscape

![IMG_20200223_190524_674](https://user-images.githubusercontent.com/61384800/75115430-7f943780-566f-11ea-8bf9-2c2ee8bb6a06.jpg)

**Comment by PDWarrior** [comment]
![IMG_20200107_224602_054](https://user-images.githubusercontent.com/61384800/75115527-4f00cd80-5670-11ea-95c3-5030ff938cdd.jpg)

**Comment by PDWarrior** [comment]
![IMG_20191122_215245_644](https://user-images.githubusercontent.com/61384800/75115565-b585eb80-5670-11ea-9e89-ed1604363181.jpg)
